sys-fs/mtd-utils/mtd-utils-2.1.2.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils vcs-snapshot
```

Comment:
Is *vcs-snapshot* really needed here?

Reply:
Yes if using http://git.infradead.org/mtd-utils.git repository. I found https://github.com/sigma-star/mtd-utils, I can change the ebuild to use that repo

Reply:
Done